### PR TITLE
Add missing dfsr metric descriptions

### LINKED ISSUE #707: dfsr collector returned error
Hello, 

After build windows_exporter.exe from the source code and run it with dfsr collector enabled, it returned below error  from the metric output:
----quote for the error-------
An error has occurred while serving metrics:

3 error(s) occurred:
* collected metric windows_dfsr_received_files_total label:<name:"name" value:"dfsr-test-{C9BCA8F8-76DB-4B79-BF13-6AD6D5CF8C31}" > counter:<value:1 >  has help "" but should have "Total number of files receieved for connection"
* collected metric windows_dfsr_rdc_received_files_total label:<name:"name" value:"dfsr-test-{C9BCA8F8-76DB-4B79-BF13-6AD6D5CF8C31}" > counter:<value:0 >  has help "" but should have "Total number of Remote Differential Compression files received"
* collected metric windows_dfsr_files_received_bytes_total label:<name:"name" value:"dfsr-test-{C9BCA8F8-76DB-4B79-BF13-6AD6D5CF8C31}" > counter:<value:435 >  has help "" but should have "Total size of files received, in bytes"
-----unquote for the error---------------------------
Could you help to check what's the issue?

### PR DESCRIPTION
**Pull request by breed808**
Reviewing these metrics has made me realise how sloppy my initial implementation was :disappointed:

@bds006 this should resolve your error with the empty metric descriptions, but keep in mind these metrics are still untested so there may be other bugs for you to discover.

Resolves #707

### CODE REVIEW

File: collector/dfsr.go [hunk 1]
```diff
@@ -124,8 +124,8 @@ func NewDFSRCollector() (Collector, error) {
 
 		// Connection
 		ConnectionBandwidthSavingsUsingDFSReplicationTotal: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "bandwidth_savings_using_dfs_replication_bytes_total"),
-			"Total amount of bandwidth savings using DFS Replication for this connection, in bytes",
+			prometheus.BuildFQName(Namespace, subsystem, "bandwidth_savings_using_dfs_replication_percentage"),
```
Comment (carlpett) [inline collector/dfsr.go:127]: I have some vague memory of this being changed *to* bytes from percentages in the original PR? Exposing the bytes if possible would be more useful, IMO (and most frequently, that is what we actually get in the data?)

File: collector/dfsr.go [hunk 2]
```diff
@@ -153,21 +153,21 @@ func NewDFSRCollector() (Collector, error) {
...
-			"",
+			"Total bytes received on the connection while replicating files using Remote Differential Compression",
 			[]string{"name"},
 			nil,
 		),
 
 		ConnectionRDCCompressedSizeOfFilesReceivedTotal: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "rdc_compressed_size_of_files_received_total"),
-			"",
+			prometheus.BuildFQName(Namespace, subsystem, "rdc_compressed_size_of_received_files_total"),
```
Comment (carlpett) [inline collector/dfsr.go:162]: This would be `_bytes_total`, right?

File: collector/dfsr.go [hunk 3]
```diff
@@ -188,57 +188,57 @@ func NewDFSRCollector() (Collector, error) {
 
 		// Folder
 		FolderBandwidthSavingsUsingDFSReplicationTotal: prometheus.NewDesc(
-			prometheus.BuildFQName(Namespace, subsystem, "bandwidth_savings_using_dfs_replication_total"),
-			"",
+			prometheus.BuildFQName(Namespace, subsystem, "bandwidth_savings_using_dfs_replication_percentage"),
```
Comment (carlpett) [inline collector/dfsr.go:191]: Same percentage/bytes question as above here

**Review by carlpett — COMMENTED** [review]
Some small nits, otherwise LGTM! :+1:

File: collector/dfsr.go (hunk 2 above)
Comment (breed808) [reply collector/dfsr.go]: Good spot, I've amended that metric.

File: collector/dfsr.go (hunk 1 above)
Comment (breed808) [reply collector/dfsr.go]: I think you're right: I've reverted the change to these two bandwidth metrics.

**Review by carlpett — APPROVED** [review]
:tada:

**Pull request merged**